provider.tf

```diff
@@ -1,4 +1,4 @@
 provider "aws" {
   region  = "${var.aws_region}"
-  version = "~> 1.6"
+  version = "~> 1.54"
```

Comment:
Hi @smelchior 

Is there a reason why the provider version check was changed from 1.6 to 1.54?

Kind regards,
Graham

Reply:
Yes, the required resource has been included in that version:
https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1540-december-21-2018

Reply:
Looks good, strange that we had 1.6 in there...